helm/adelphi/templates/cassandra-image-builder.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: cassandra-image
+  namespace: {{ .Values.namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
+spec:
+  templates:
+  - name: cassandra-image-template
+    inputs:
+      parameters:
+      - name: registry_ip
+      - name: commit_hash
+    container:
+      image: gcr.io/kaniko-project/executor:v1.0.0
+      args:
+      - --dockerfile=/dockerfile/Dockerfile
+      - --context=dir:///workspace
+      - {{`--destination={{inputs.parameters.registry_ip}}:30000/cassandra-quality/cassandra`}}
```

Comment:
Don't we want this to be "adelphi" rather than "cassandra-quality"?

Reply:
Fixed in the follow-up PR #35